source/monkeyPatches/enumPatches.py

```diff
@@ -0,0 +1,60 @@
+# A part of NonVisual Desktop Access (NVDA)
+# Copyright (C) 2021 NV Access Limited
+# This file is covered by the GNU General Public License.
+# See the file COPYING for more details.
+
+
+def replace__new__():
+	import enum
+	# prevent cyclic references on ValueError during construction

```

Comment:
Have you tested if this occurs in more recent versions of Python? If it isn't fixed it might be worth reporting this issue against them.

Reply:
The latest implementation in cpython is even more complicated. From inspection it does appear to have the same flaw.  I will be creating an issue / PR to fix it.